<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>396</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>404</width>
    <height>396</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UTProfile</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>UVs</string>
       </attribute>
       <widget class="QWidget" name="verticalWidget" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
         <item>
          <widget class="QListView" name="listUV"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabForm">
       <attribute name="title">
        <string>Formations</string>
       </attribute>
       <widget class="QWidget" name="verticalWidgetF" native="true">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>50</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3"/>
       </widget>
       <widget class="QListView" name="listFormation">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>161</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabCat">
       <attribute name="title">
        <string>Catégories</string>
       </attribute>
       <widget class="QWidget" name="verticalWidget_2" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>141</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
         <item>
          <widget class="QListView" name="listCategorie"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabEtu">
       <attribute name="title">
        <string>Étudiant</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>404</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>UVs</string>
    </property>
    <addaction name="actionCatalogue_UV"/>
   </widget>
   <widget class="QMenu" name="menuEtudiant">
    <property name="title">
     <string>Etudiant</string>
    </property>
    <addaction name="actionCharger_un_profil"/>
    <addaction name="actionNouveau_profil"/>
    <addaction name="separator"/>
    <addaction name="actionEnregistrer_le_profil"/>
    <addaction name="actionEnregistrer_sous"/>
   </widget>
   <widget class="QMenu" name="menuApplication">
    <property name="title">
     <string>Application</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuApplication"/>
   <addaction name="menuFichier"/>
   <addaction name="menuEtudiant"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionCatalogue_UV">
   <property name="text">
    <string>Modifier une UV</string>
   </property>
  </action>
  <action name="actionUV">
   <property name="text">
    <string>UV</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionCharger_un_profil">
   <property name="text">
    <string>Charger depuis un fichier...</string>
   </property>
  </action>
  <action name="actionNouveau_profil">
   <property name="text">
    <string>Nouveau profil...</string>
   </property>
  </action>
  <action name="actionEnregistrer_le_profil">
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </action>
  <action name="actionEnregistrer_sous">
   <property name="text">
    <string>Enregistrer dans un fichier...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCatalogue_UV</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openUV()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUV</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openUV()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNouveau_profil</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>cbNouveauProfil()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEnregistrer_le_profil</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>cbProfilSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEnregistrer_sous</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>cbProfilSaveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCharger_un_profil</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>cbLoadProfil()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openChargerUV()</slot>
  <slot>openUV()</slot>
  <slot>cbNouveauProfil()</slot>
  <slot>cbProfilSave()</slot>
  <slot>cbProfilSaveAs()</slot>
  <slot>cbLoadProfil()</slot>
 </slots>
</ui>
